api/src/controllers/team.js

```diff
@@ -247,22 +244,22 @@ async function deployToGHPages(req, res, next) {
  * @returns 404: team is not found
  * @returns 400: team id is not in a valid hexadecimal format
  */
-function updateTeam(req, res, next) {
+// eslint-disable-next-line no-unused-vars
```

Comment:
nit:this is my personal preference, but consider inlining this eslint message to the same line as the function definition (L248). See: https://github.com/Researchify/Researchify/blob/main/api/src/middleware/error.js#L37